app/assets/stylesheets/_sponsor_application.scss

```diff
@@ -78,3 +78,75 @@
 .steps.active:after {
   background: rgba(0, 0, 0, 0.5);
 }
+
+.status{
+  margin: 20px 0;
+  clear: both;
+}
+
+.status__heading{
+  line-height: 25px;
```

Comment:
Properties should be ordered float, font-weight, line-height, margin, margin-bottom, margin-left